build/BuildFunctions.psm1

```diff
@@ -150,20 +150,17 @@ function Invoke-MochaTest([string]$task, [string]$reporter) {
 
 function Send-Coverage() {
     Write-Host "Sending Code Coverage reports."
-    $reports = ls -Recurse -Include coverage-final.json
-    $reports.FullName | % {
-        # codecov uploads code coverage reports to codecov.io.
-        Write-Verbose "Running: codecov -f $_"
-        codecov -f $_
-    }
+    $env:PATH = "C:\msys64\usr\bin;$env:PATH"
```

Comment:
Is the path `C:\msys64\usr\bin` always the same?

Reply:
Yes.